mkdocs/livereload/__init__.py

```diff
@@ -70,9 +70,13 @@ def watch(self, path, func=None, recursive=True):
             )
 
         def callback(event):
-            with self._rebuild_cond:
-                self._to_rebuild[func] = True
-                self._rebuild_cond.notify_all()
+            # Text editors always cause a "file close" event in addition to "modified" when saving a
+            # file. Some editors also have "swap" functionality that keeps writing into another file
+            # that's never closed. Prevent such write events from causing a rebuild.
```

Comment:
Agh I think `watchdog` emits "file close" events only on Linux 
so what I'm describing and attempting in this last commit cannot work :weary: 

Why I tried this: https://github.com/mkdocs/mkdocs/issues/2061#issuecomment-837435554

Reply:
Applied solution: https://github.com/gorakhargosh/watchdog/issues/795#issuecomment-839950250